ddtrace/tracer/textmap_test.go

```diff
@@ -242,6 +242,26 @@ func TestB3(t *testing.T) {
 		assert.Equal(sctx.spanID, uint64(1))
 	})
 
+	t.Run("extract_32", func(t *testing.T) {
```

Comment:
Instead of duplicating the previous test, wouldn't it be easier and provide better coverage to modify the previous one and convert it to a [table driven test](https://github.com/golang/go/wiki/TableDrivenTests) ?

Reply:
Agree, good suggestion. What about this latest update?